service/controller/app/v1/resource/chartstatus/create.go

```diff
@@ -0,0 +1,49 @@
+package chartstatus
```

Comment:
```suggestion
package status
```

Lets just call this `status` like in chart-operator. It sets the status for the app CRs.